cfe_internal/enterprise/CFE_hub_specific.cf

```diff
@@ -430,6 +415,25 @@ bundle agent log_cfengine_enterprise_license_utilization
 
 }
 
+bundle agent cfe_internal_enterprise_HA_classes
+{
+  classes:
+      # NOTE The `hub_active` class is a hard class defined by the ha_plugin in
+      # the enterprise agent.
+      "active_hub"
+        expression => "policy_server.(!enable_cfengine_enterprise_hub_ha|(enable_cfengine_enterprise_hub_ha.hub_active))",
```

Comment:
Maybe drop a comment here, I started to question why were were defining active_hub when we have hub_active already. Reading the expression carefully helped me realize that the intent here is to identify the active hub as the curreently active ha hub, or just a normal hub. No biggie, lgtm.

Reply:
Added.